app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ProjectsController < ActionController::Base
```

Comment:
This line should be inheriting from `ApplicationController` not `ActionController::Base`. This is probably where your error was coming from.